futures-io/src/lib.rs

```diff
@@ -362,7 +362,7 @@ if_std! {
         ) -> Poll<Result<usize>> {
             let position = self.position();
             let result = {
-                let mut out = self.get_mut().as_mut();
+                let out = self.get_mut().as_mut();
```

Comment:
Can you explain why this `mut` is unused? It looks used to me (there's an `&mut out` below). Is it just because it's an `&mut` already and some coercion is firing?

Reply:
Got it, thanks.